webdriver/tests/user_prompts/send_alert_text.py

```diff
@@ -0,0 +1,67 @@
+from tests.support.asserts import assert_error, assert_success
+
+
+def send_alert_text(session, body=None):
+    return session.transport.send("POST", "session/{session_id}/alert/text"
+                                  .format(session_id=session.session_id), body)
+
+# Used to check if alert text sent was correct
+def accept_alert(session):
+    return session.transport.send("POST", "session/{session_id}/alert/accept"
+                                  .format(session_id=session.session_id))
+
+
+# 18.4 Send Alert Text
+
+def test_no_browsing_context(session, create_window):
+    # 18.4 step 1
+    session.window_handle = create_window()
+    session.close()
+
+    body = {
+        "text": "Federer"
+        }
```

Comment:
Can you please pull all the bodies throughout the test onto a single line?

Reply:
fixed